<?xml version="1.0"?>

<launch>
	<group ns="robot4">
		<arg name="robot_node_4" default="true" />
        
		<node pkg="robot_controller" type="robot" name="robot_node_4" if="$(arg robot_node_4)" output="screen">
			<param name ="trajectory_file" value = "$(find robot_controller)/launch/Orbital_ECI.txt"/>
			<param name ="robot_name" value = "robot4"/>
            <param name ="kd" value = "-0.2"/>
            <param name ="kp" value = "-0.6"/>
            <param name ="ku" value = "-0.001"/>
            <param name ="angular_speed_max" value = "0.1"/>
            <param name ="xy_speed_constraint" value = "0.1"/>
		</node>
	</group>
</launch>
